ts-apps/webview/vite.config.ts

```diff
@@ -8,5 +8,8 @@ export default defineConfig({
     rollupOptions: {
       external: ['vscode-webview'],
     },
+    // While we're still in alpha: Make it easier to debug
```

Comment:
Misc refactor / infra: Disable minifying (and enable sourcemap) in vscode webview's vite config to facilitate debugging (we can change this once we have a serious 'prod mode' for VSCode)